src/Game.coffee

```diff
@@ -243,11 +244,20 @@ class Game
     @sendMessage redraw, @, @
     @score += POINTS_PER_COIN
     @updateScore()
+    # perfectly counts and updates remaining coins on the board
+    #@board.revealPitTiles()
+    console.log("Coins Remaining: #{@board.coinsRemaining()}")
     if @board.coinsRemaining() <= 0
-      @board.revealAll()
-      @sendMessage redraw, @, @
-      # TODO - [rmarks] reset the board
+      @board.revealPitTiles()
+      @timer.pauseTimer = true
+      @board.pauseCollectCoins = true
+      setTimeout( (() =>
```

Comment:
I think that this code is not quite `coffeescript`ish enough
something more along these lines reads better in my opinion
```coffee
onTimeout = ->
  @board.reset()
  @input.enable()
  @timer.resume()
onTimeout = onTimeout.bind @
setTimeout onTimeout, 1500
```